src/main/java/org/janelia/saalfeldlab/paintera/viewer3d/ViewFrustumCulling.java

```diff
@@ -120,13 +112,12 @@ public AffineTransform3D getTransform()
 	 * @param block
 	 * @return
 	 */
-	public boolean intersects(final RealInterval block)
-	{
+	public boolean intersects(final RealInterval block) {
 		// sphere-based test
 		final double[] sphereCenterPos = {
-			(block.realMin(0) + block.realMax(0)) / 2,
-			(block.realMin(1) + block.realMax(1)) / 2,
-			(block.realMin(2) + block.realMax(2)) / 2
+				(block.realMin(0) + block.realMax(0)) / 2,
```

Comment:
indentation?

Reply:
This is also from the initial Java 13 Migration commit